renderer/views/torrent-list.js

```diff
@@ -153,7 +153,7 @@ function TorrentList (state) {
         <i.button-round.icon.play
```

Comment:
ya, i thought it was a bit confusing since the function it exports is still called TorrentList. Then in `app.js`, it used to say `const TorrentList = require('home')`. I think TorrentList is nice and descriptive